sambaplayersdk/src/main/java/com/sambatech/player/SambaPlayer.java

```diff
@@ -955,11 +977,13 @@ public void onClick(View v) {
 				});
 
 		player.setOutputMenu(outputMenu);
+		player.setControlsVisible(true, Controls.OUTPUT);
 	}
 
 	private void initCaptionMenu() {
-		if (media.isAudioOnly || media.captions == null ||
-				media.captions.size() == 0 || player == null)
+		if (player == null || media.isAudioOnly ||
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.qa.sambatech.com/sonar/coding_rules#rule_key=squid%3AS1067)